build/rakelib/cluster.rake

```diff
@@ -119,6 +119,42 @@ namespace :cluster do
     end
   end
 
+  desc "Deploy ebrelayer"
+  namespace :ebrelayer do
+    task :deploy, [:chainnet, :provider, :namespace, :image, :image_tag, :ethereum_websocket_address, :ethereum_contract_address, :ethereum_private_key] do |t, args|
+      check_args(args)
+
+      cmd = %Q{helm upgrade sifnode ../build/helm/sifnode \
+        --set sifnode.env.chainnet=#{args[:chainnet]} \
+        --install -n #{ns(args)} \
+        --set image.tag=#{image_tag(args)} \
+        --set image.repository=#{image_repository(args)} \
+        --set ebrelayer.enabled=true \
+        --set ebrelayer.image.tag=#{image_tag(args)} \
+        --set ebrelayer.env.ethereumWebsocketAddress=#{args[:ethereum_websocket_address]} \
+        --set ebrelayer.env.ethereumContractAddress=#{args[:ethereum_contract_address]} \
+        --set ebrelayer.env.moniker=#{args[:chainnet]} \
```

Comment:
The moniker is not the same as the chain name. We should have a specific argument for moniker.